File: src/main/java/de/bnder/taskmanager/commands/Help.java
package de.bnder.taskmanager.commands;

import de.bnder.taskmanager.main.Command;
import de.bnder.taskmanager.utils.Localizations;
import de.bnder.taskmanager.utils.MessageSender;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

import java.awt.*;
import java.io.IOException;
import java.util.ArrayList;

public class Help implements Command {
    @Override
    public void action(String[] args, GuildMessageReceivedEvent event) throws IOException {
        final String prefix = String.valueOf(event.getMessage().getContentRaw().charAt(0));
        final String langCode = Localizations.getGuildLanguage(event.getGuild());
        final String embedTitle = Localizations.getString("help_message_title", langCode);
        MessageSender.send(embedTitle + " - " + Localizations.getString("group_title", langCode), Localizations.getString("help_message_group_commands", langCode, new ArrayList<String>(){{
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
        }}), event.getMessage(), Color.cyan);
        MessageSender.send(embedTitle + " - " + Localizations.getString("task_message_title", langCode), Localizations.getString("help_message_task_commands", langCode, new ArrayList<String>(){{
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
        }}), event.getMessage(), Color.cyan);
        MessageSender.send(embedTitle, Localizations.getString("help_message_other_commands", langCode, new ArrayList<String>(){{
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
        }}), event.getMessage(), Color.cyan);
        MessageSender.send(embedTitle + " - " + Localizations.getString("permissions_title", langCode), Localizations.getString("help_message_permission_commands", langCode, new ArrayList<String>(){{
            add(prefix);
            add(prefix);
            add(prefix);
            add(prefix);
        }}), event.getMessage(), Color.cyan);
    }
}

File: src/main/java/de/bnder/taskmanager/commands/Help.kt
package de.bnder.taskmanager.commands

import de.bnder.taskmanager.main.Command
import de.bnder.taskmanager.utils.Localizations
import de.bnder.taskmanager.utils.MessageSender
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
import java.awt.Color

/*
 * Copyright (C) 2019 Jan Brinkmann
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
class Help : Command {
    override fun action(args: Array<String>, event: GuildMessageReceivedEvent) {
        val langCode = Localizations.getGuildLanguage(event.guild)
        MessageSender.send(Localizations.getString("help_message_title", langCode), Localizations.getString("help_message_group_commands", langCode) + Localizations.getString("help_message_task_commands", langCode) + Localizations.getString("help_message_other_commands", langCode), event.message, Color.cyan)
    }
}